components/date-picker/locale/es_ES.ts

```diff
@@ -8,9 +8,28 @@ const locale: PickerLocale = {
   lang: {
     placeholder: 'Seleccionar fecha',
     rangePlaceholder: ['Fecha inicial', 'Fecha final'],
+    shortWeekDays: ['Dom', 'Lun', 'Mar', 'Mie', 'Jue', 'Vie', 'Sab'],
+    yearFormat: 'YYYY',
+    dateFormat: 'DD/MM/YYYY',
+    fieldDateTimeFormat: 'DD/MM/YYYY HH:mm:ss',
+    shortMonths: [
+      'Ene',
+      'Feb',
+      'Mar',
+      'Abr',
+      'May',
+      'Jun',
+      'Jul',
+      'Ago',
+      'Sep',
+      'Oct',
+      'Nov',
+      'Dic',
+    ],
     ...CalendarLocale,
   },
   timePickerLocale: {
+    placeholder: 'Seleccionar hora',
```

Comment:
Move it into `components/time-picker/locale/es_ES.ts`

Reply:
I didn't realize that file was there. It's fixed now! [Here.](https://github.com/ant-design/ant-design/pull/48601/commits/7102467d410b0f7b70d96c7fe8e3a5613d20b69a)